docs/sql-ref-syntax-qry-select-clusterby.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: global
+title: CLUSTER BY Clause
+displayTitle: CLUSTER BY Clause
+license: |
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+ 
+     http://www.apache.org/licenses/LICENSE-2.0
+ 
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+---
+The <code>CLUSTER BY</code> clause is used to first repartition the data based
+on the input expressions and then sort the data within each partition. This is
+semantically equivalent to performing <code>DISTRIBUTE BY</code> followed by
+<code>SORT BY</code>. This cause only ensures that the resultant rows are
+sorted within each partition and does not guarantee total ordering of output.
+
+### Syntax
+{% highlight sql %}
+CLUSTER BY { expression [ , ...] }
+{% endhighlight %}
+
+### Parameters
+<dl>
+  <dt><code><em>expression</em></code></dt>
+  <dd>
+    Specifies combination of one or more values, operators and SQL functions that results in a value.
+  </dd>
+</dl>
+
+### Examples
+{% highlight sql %}
+CREATE TABLE person (name STRING, age INT);
+INSERT INTO person VALUES ('Zen Hui', 25), 
```

Comment:
Do you have any reference for this format? I checked the PgSQL document and they have the other form like this;
```
INSERT INTO person VALUES
    ('Zen Hui', 25), 
    ('Anil B', 18), 
    ('Shone S', 16), 
    ('Mike A', 25),
    ('John A', 18), 
    ('Jack N', 16);
```
https://www.postgresql.org/docs/current/sql-insert.html#id-1.9.3.152.9

Reply:
@maropu No.. i don't :-) Lets me fix this for all the prs thats open currently. I will fix the one that are closed in the finalization pr.